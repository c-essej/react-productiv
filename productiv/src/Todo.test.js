import Todo from './Todo';
import React from "react"
import { render } from "@testing-library/react";
import TEST_TODOS from './_testCommon'

const TEST_TODO = TEST_TODOS[0]

it("renders without crashing", function () {
  render( <Todo todo={TEST_TODO}/>)
});

it("matches snapshot", function () {
  const { container } = render(
    <Todo todo={TEST_TODO} />);

expect(container).toMatchSnapshot();
});


