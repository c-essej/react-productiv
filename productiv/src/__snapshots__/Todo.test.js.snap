// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<div>
  <div
    class="Todo"
  >
    <div>
      <b>
        test1
      </b>
       
      <small>
        Priority 1
      </small>
    </div>
    <div>
      <small>
        test1 description
      </small>
    </div>
  </div>
</div>
`;
